.github/workflows/build-and-test.yml

```diff
@@ -115,8 +115,8 @@ jobs:
       - name: Lint
         run: |
           make install-tools
-          make -j4 for-all CMD="make checklicense impi lint misspell"
-  
+          make -j4 for-all-target TARGET="checklicense impi lint misspell"
```

Comment:
Testing this out I think `ALL_GO_MODULES` doesn't include the root directory because of the final `^./` filter so we may be unintentionally losing coverage for the project?

Reply:
ah that might explain why it's faster ;) checking.

Reply:
good catch! Fixed.